index.js

```diff
@@ -17,7 +17,12 @@ var WinstonCloudWatch = function(options) {
   var awsSecretKey = options.awsSecretKey;
   var awsRegion = options.awsRegion;
   var messageFormatter = options.messageFormatter ? options.messageFormatter : function(log) {
-    return [ log.level, log.msg, stringify(log.meta) ].join(' - ');
+    if(stringify(log.meta)==="{}")
```

Comment:
What about `_.isEmpty`?
Also could you please expand on why `meta` is necessary in addition to `msg`? (I am not sure I got that so I'm missing something here).